#firmware
string Generic_Node_Name
string Node_Name
uint8 Major_Release
uint8 Minor_Release
uint8 Build_Number
string Description
